project.clj

```diff
@@ -84,13 +84,13 @@
                          [trptcolin/versioneer "0.2.0"]
                          [io.dropwizard.metrics/metrics-core ~dropwizard-metrics-version]
                          [io.dropwizard.metrics/metrics-graphite ~dropwizard-metrics-version]
-                         [metrics-clojure "2.6.1"]
+                         [metrics-clojure "2.10.0"]
                          [org.ow2.asm/asm-all "5.0.3"]
-                         [honeysql "0.6.3"]
-                         [org.postgresql/postgresql "42.2.0"]
+                         [honeysql "0.9.2"]
```

Comment:
this lists multiple breaking changes in its changelog and needs to be checked by the teams that use it (I know at least of puppetdb)

Reply:
I can revert it.  Thoughts?

Reply:
I'll check it against the orchestrator too, which uses honeysql